CONTRIBUTING.md

```diff
@@ -104,9 +104,9 @@ Crayons repository uses github pre commit hooks, follow the below steps to commi
 
 8. Give a commit message when it prompts.
```

Comment:
Could we modify step 7 and 8 as follows?

- 7. Fill in the following details when prompted.
- 8. Give a commit message when prompted.

Reply:
@menakallg, do you want to start using roman numerals with step 7 and 8 only?

Reply:
No. GitHub auto-corrected the numbering. :-)